pkg/api/v1alpha1/kepler_internal_types.go

```diff
@@ -53,8 +54,71 @@ type OpenShiftSpec struct {
 
 // KeplerInternalSpec defines the desired state of KeplerInternal
 type KeplerInternalSpec struct {
-	Exporter  InternalExporterSpec `json:"exporter"`
-	OpenShift OpenShiftSpec        `json:"openshift,omitempty"`
+	Exporter    InternalExporterSpec     `json:"exporter"`
+	Estimator   *InternalEstimatorSpec   `json:"estimator,omitempty"`
+	ModelServer *InternalModelServerSpec `json:"modelServer,omitempty"`
+	OpenShift   OpenShiftSpec            `json:"openshift,omitempty"`
+}
+
+// Kepler Model Server Spec
+type InternalModelServerSpec struct {
+
+	// +kubebuilder:default=true
```

Comment:
could you please keep the default to `false` for all features that are not part of stable api - `kepler` ?

Reply:
Done. Thanks.